src/11_args.py

```diff
@@ -44,7 +53,9 @@
 # Note: Google "python keyword arguments".
 
 # YOUR CODE HERE
-
+def f4(**kwargs):
+    for key in kwargs:
```

Comment:
A cool thing you can do in python:
`def f4(**kwargs):
  for k, v in kwargs.items():
    print(f'key: {k}, value: {v}')`